processor/configuration/src/main/resources/schemas/sources/azure_eventhubs_source_0.1.json

```diff
@@ -0,0 +1,69 @@
+{
+  "type" : "object",
+  "additionalProperties" : false,
+  "required" : [
```

Comment:
Why is `kafka_topic` not included? 

It's [in](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/blob/037b5e1da16d31513fba7fe29b153c536255f955/etc/kubernetes/manifests/base/connectors/connector-catalog-camel-azure/azure_eventhubs_source_0.1.json#L45) the COS [definition](https://github.com/bf2fc6cc711aee1a0c2a/cos-fleet-catalog-camel/blob/037b5e1da16d31513fba7fe29b153c536255f955/etc/kubernetes/manifests/base/connectors/connector-catalog-camel-azure/azure_eventhubs_source_0.1.json#L101).

A minor nit pick too; if this is a copy and paste of the definition in COS why are the properties in a different order? (It just makes comparing them more difficult than it needs to be).

Reply:
we don't need input for `kafka_topic` from enduser. RHOSE have this info by itself thats why I had removed `kafla_topic` argument.
sorry for ordering. I make it fix.

Reply:
I had fixed the ordering

Reply:
@manstis The `kafka_topic` setting controls the kafka topic that the connector should use to consume messages. As this is an internal detail - the user does not have to provide his/her kafka cluster/topic - we don't include it in the json schema (if the user tries to submit it, the validation fails) and then we inject it in the connector's creation payload here https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/27b972abf4f915081b4e4aae8e7ea902487b0832/processor/configuration/src/main/java/com/redhat/service/smartevents/processor/GatewayConnector.java#L47 using the kafka topic created by the manager